lib/bigcommerce/resources/customers/customer.rb

```diff
@@ -26,5 +28,21 @@ class Customer < Resource
     def self.count(params = {})
       get 'customers/count', params
     end
+
+    # Generate a token that can be used to log the customer into the storefront.
+    # This requires your app to have the store_v2_customers_login scope and to
+    # be installed in the store.
+    def login_token
```

Comment:
In order to protect the thread safety of this, we should allow a user to pass in the store_hash, client_id, and client_secret (AKA a config)